pkgs/development/python-modules/deepdiff/default.nix

```diff
@@ -48,10 +49,15 @@ buildPythonPackage rec {
     pyyaml
   ];
 
+  disabledTests = [
+    # Assertion issue with the decimal places
+    "test_get_numeric_types_distance"
```

Comment:
The tests all succeeded for me, why did you disable this one?

Reply:
So I guess this diff hunk will be the ideal for 21.11: https://github.com/NixOS/nixpkgs/pull/173574